### PR TITLE
readme: Add links to CHANGELOG.md and LICENSE

### PR DESCRIPTION
**Pull request by wbnns**
For convenience, added links to the `CHANGELOG.md` and `LICENSE` that are referenced in `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,10 +101,10 @@ support only a single blockchain.
 
 # Release Notes
 
-See CHANGELOG.md
+See [CHANGELOG.md](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/CHANGELOG.md).
```
Comment (dongcarl) [inline README.md]:
```suggestion
See [CHANGELOG.md](CHANGELOG.md).
```

File: README.md [hunk 2]
```diff
@@ -101,10 +101,10 @@ support only a single blockchain.
...
 
 
 # Licensing
 
-The code in this project is licensed under the Creative Commons CC0 1.0
-Universal license.
+The code in this project is licensed under the [Creative Commons CC0 1.0
+Universal license](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/LICENSE).
```
Comment (dongcarl) [inline README.md]:
```suggestion
Universal license](LICENSE).
```

**Comment by stevenroose** [comment]
+1 for relative links

**Comment by wbnns** [comment]
Thanks! Done (410e45f378f5e90d58d072c86f968844747c295d).

**Comment by elichai** [comment]
FYI CI failures are because of https://github.com/rust-bitcoin/rust-bitcoin/pull/398

**Comment by stevenroose** [comment]
Yeah let's get #398 in asap :)

**Pull request merged**